TelCo.ColorCoder/ColorPair.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class ColorPair
+    {
+        public Color majorColor;
+        public Color minorColor;
+        public override string ToString()
+        {
+            return string.Format("MajorColor:{0}, MinorColor:{1}", majorColor.Name, minorColor.Name);
+        }
+        public static int GetPairNumberFromColor(ColorPair pair)
+        {
+            // Find the major color in the array and get the index
+            int majorIndex = -1;
+            for (int i = 0; i < Program.colorMapMajor.Length; i++)
```

Comment:
The name 'Program' is not clear. Is it a namespace to hold constants / configuration? Give a suitable name, according to its purpose.

Reply:
Sure. Will change the name of the class.